codelists/+partyRole.csv

```diff
@@ -5,3 +5,11 @@ processContactPoint,Process contact point,A contact point dedicated to this cont
 reviewContactPoint,Review contact point,The service from which information about the review procedure can be obtained.
 selectedParticipant,Selected participant,A party that has already been selected to participate in the design contest.
 informationService,Information service,An organization that can provide information regarding specific aspects of the contracting process.
+procurementServiceProvider,Procurement service provider,An organization that provides procurement services to contracting authorities.
+eSender,TED eSender,An organization that provides data transfer services to contracting authorities.
+leadBuyer,Lead buyer,The lead buyer in a group of buyers.
+awardingCentralPurchasingBody,Awarding central purchasing body,"A central purchasing body awarding public contracts or concluding framework agreements for goods, services or works intended for other buyers."
+acquiringCentralPurchasingBody,Acquiring central purchasing body,"A central purchasing body acquiring goods, services or works intended for other buyers."
```

Comment:
To align with the direction of https://github.com/open-contracting/standard/issues/1180 Since we're discussing adding wholesaler to OCDS 1.2, we maybe add this code in the 1.2 draft extension instead.

```suggestion
wholesaler,Wholesaler,"A buyer (e.g. a central purchasing body) of goods, services or works intended for other buyers. Organizations with the 'wholesaler' role must have the 'buyer' role only if referenced from the `buyer` field."
```

Reply:
Just a note for my own benefit that 'wholesaler' is under discussion in https://github.com/open-contracting/standard/issues/1180 so I'll wait for that discussion to be resolved before making any updates here.

Reply:
Noting that we settled on 'wholesaleBuyer'

Reply:
Added to the 1.2 draft extension in https://github.com/open-contracting-extensions/ocds_1_2_draft_extension/pull/2

Reply:
590075767965d824effbc8eb94f031fe41dcd665 should have removed 'acquiringCentralPurchasingBody', not 'awardingCentralPurchasingBody'.

We can rename 'awardingCentralPurchasingBody' to 'awardingBuyer' – or 'contractIntermediaryBuyer' (reusing language from Jáchym).

Reply:
'acquiringCentralPurchasingBody' removed in https://github.com/open-contracting-extensions/ocds_eu_extension/pull/12/commits/545ef4f0d5968f8f3e1fd4e1416eeb93c5f8fb03

Another benefit of 'wholesaleBuyer' is that we no longer have two very long codes with only a small difference!